### DISCUSSION TITLE
what might be the best workflow to move rules to another account?

### DISCUSSION THREAD

**Discussion by jquiros2** [firefly-iii/firefly-iii#7836] [Firefly III]
So I have 40+ rules I've been tending for some time, and now that I am about to onboard y wife into her own account I would like to be able to share those same rules to her account as a head start before importing transactions.  I am about to start looking into the rules in the database, and also saw that we can export the rules (--export-rules) but am wondering if there is any process for this already documented.

**Comment by JC5** [comment]
You could import them or create them using the API, but this is something you'd have to program yourself.
